src/firstpass.rs

```diff
@@ -457,276 +456,273 @@ impl<'a, 'b> FirstPass<'a, 'b> {
         let mut last_pipe_ix = start;
         let mut begin_text = start;
 
-        let (final_ix, brk) =
```

Comment:
This is unfortunately just `rustfmt` deciding that this entire expression needs to rewritten. Sorry for the noise! Although it does reduce the entire indentation level by one, which is nice. I'm just hoping it doesn't jump back on further modification...